### ISSUE TITLE
Microsoft Live Share Support

### ISSUE THREAD

**Issue by jefersonchaves** [forcedotcom/salesforcedx-vscode#2214]
**Is your feature request related to a problem? Please describe.**
Now more than even we are working remotely and many times we work in pairs. Would be very helpful to be able to push code and run the tests through test runner.

**Describe the solution you'd like**
The ability for the remote session to:
- Push code.
- View tests and to execute them.

**Describe alternatives you've considered**
Having a separate plugin to allow this to happen.
The current only workaround is to share a terminal between the sessions.

**Comment by smaddox-sf** [comment]
Hi @jefersonchaves - Thank you for submitting this idea.  I can see the usefulness and boost to productivity.  This isn't on our near-term roadmap but we'll consider it in the future.

**Comment by jefersonchaves** [comment]
Thanks @smaddox-sf, if there is something I could do to help, please don't hesitate to let me know.

**Comment by MGarfOppLoans** [comment]
Agreed, currently the biggest issue is you can't see deploy on save results while live sharing

**Comment by jefersonchaves** [comment]
Hello, @smaddox-sf, is there any chance that this can get in a roadmap from now on? I have been a while and now we most of us are remote.
